tests/sclT0009-unused-symbols/sym01.exec

```diff
@@ -0,0 +1,2 @@
+# format: exec
+exec:
```

Comment:
Yes, but the tests are failing now on Travis, although they are running fine locally :(
I'm not sure what the problem is...